platformio.ini

```
@@ -13,13 +13,19 @@ platform = https://github.com/pioarduino/platform-espressif32.git#53.03.10
 #framework = arduino, espidf
 framework = espidf
 monitor_speed = 115200
-monitor_filters = direct
+monitor_filters = 
+    direct
+    ; esp32_exception_decoder 
 board_build.embed_txtfiles =
     managed_components/espressif__esp_insights/server_certs/https_server.crt
     managed_components/espressif__esp_rainmaker/server_certs/rmaker_mqtt_server.crt
     managed_components/espressif__esp_rainmaker/server_certs/rmaker_claim_service_server.crt
     managed_components/espressif__esp_rainmaker/server_certs/rmaker_ota_server.crt
-
+build_type = debug
+board_build.partitions = partitions.csv
+; build_flags = 
+;  -D CORE_DEBUG_LEVEL=5
+  
 [env:esp32dev]
 board = esp32dev
```

If we move build_type debug to it's own section, we can switch between debug and release faster
```suggestion
build_type = release
  
[env:esp32dev]
board = esp32dev

[env:esp32dev-debug]
board = esp32dev
build_type = debug
board_build.partitions = partitions.csv
; build_flags = 
;  -D CORE_DEBUG_LEVEL=5
```